spikeinterface/postprocessing/tests/test_correlograms.py

```diff
@@ -20,30 +23,119 @@
     cache_folder = Path("cache_folder") / "postprocessing"
 
 
-def _test_correlograms(sorting, window_ms: float, bin_ms: float, methods: List[str]):
+def test_make_bins():
+    sorting = generate_sorting(num_units=5, sampling_frequency=30000., durations=[10.325, 3.5])
+    
+    window_ms = 43.57
+    bin_ms = 1.6421
+    bins, window_size, bin_size = _make_bins(sorting, window_ms, bin_ms)
+    assert bins.size == np.floor(window_ms / bin_ms) +1 
+    # print(bins, window_size, bin_size)
+
+    window_ms=60.0
+    bin_ms=2.0
+    bins, window_size, bin_size = _make_bins(sorting, window_ms, bin_ms)
+    assert bins.size == np.floor(window_ms / bin_ms) +1 
+    # print(bins, window_size, bin_size)
+    
+
+def _test_correlograms(sorting, window_ms, bin_ms, methods):
     for method in methods:
         correlograms, bins = compute_correlograms(sorting, window_ms=window_ms, bin_ms=bin_ms, symmetrize=True, 
                                                   method=method)
-
         if method == "numpy":
             ref_correlograms = correlograms
             ref_bins = bins
         else:
+            import matplotlib.pyplot as plt
+            
+            for i in range(ref_correlograms.shape[1]):
+                fig, ax = plt.subplots()
+                ax.plot(bins[:-1], ref_correlograms[0, i, :], color='green', label='numpy')
+                ax.plot(bins[:-1], correlograms[0, i, :], color='red', label=method)
+                ax.legend()
+
+                plt.show()
             assert np.all(correlograms == ref_correlograms), f"Failed with method={method}"
             assert np.allclose(bins, ref_bins, atol=1e-10), f"Failed with method={method}"
 
-@pytest.mark.skip(reason="Is going to be fixed (PR #750)")
-def test_compute_correlograms():
-    methods = ["numpy", "auto"]
+def test_equal_results_correlograms():
+    # compare that the 2 methods have same results
+    methods = ["numpy"]
     if HAVE_NUMBA:
         methods.append("numba")
 
-    recording, sorting = se.toy_example(num_segments=2, num_units=10, duration=100)
+    sorting = generate_sorting(num_units=5, sampling_frequency=30000., durations=[10.325, 3.5])
```

Comment:
I noticed that if the duration is too low, you can miss some differences (because there can be some differences for an exact timing).
Maybe bring the number of units and/or the duration up?

Reply:
> Why short duration would make the test fails mote than long duration.

You can have a difference between 2 methods that is missed by a short duration (due to insufficient spikes). So longer duration / higher number of units is usually better (but takes more time).